src/pool/srv_target.c

```diff
@@ -88,6 +88,21 @@ pool_child_lookup_noref(const uuid_t uuid)
 	return NULL;
 }
 
+struct ds_pool_child *
+ds_pool_child_find(const uuid_t uuid)
```

Comment:
This is a improper name, I'd rather call it ds_pool_child_lookup_force(), it's confusing to have both find() and lookup()